frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -305,8 +301,10 @@ final class BloopBspServices(
               compileProvider = Some(BloopBspServices.DefaultCompileProvider),
               testProvider = Some(BloopBspServices.DefaultTestProvider),
               runProvider = Some(BloopBspServices.DefaultRunProvider),
+              debugProvider = None, // todo kpodsiad
```

Comment:
Turn it into a issue?

Reply:
heh - this could be easily fixed - created #1781